### PR TITLE
[Legacy] V3

### PR DESCRIPTION
**Pull request by vadimdemedes**

### CODE REVIEW

File: lib/query.js [hunk 1]
```diff
@@ -1,530 +1,142 @@
...
-	this.query[key] = value;
+		let select = {};
 
-	return this;
-};
+		if (isObject(field)) {
+			select = field;
+		}
 
+		select[field] = value;
```
Comment (zaaack) [inline lib/query.js]: Is this a bug? If field is an object, it shouldn't be select's key

File: lib/query.js [hunk 2]
```diff
@@ -1,530 +1,142 @@
...
-					doc[key] = orderedDocuments;
-				});
-			}).then(function () {
-				return new Model(doc, {
-					populate: options.populate
-				});
-			});
-		});
-};
+		select[field] = value;
```
Comment (zaaack) [inline lib/query.js]: Same here.

File: lib/query.js (hunk 1 above)
Comment (vadimdemedes) [inline lib/query.js]: Fixed, thank you!

**Comment by vadimdemedes** [comment]
Closing this PR in favor of https://github.com/vadimdemedes/mongorito/pull/169, but leaving branch alive, because some people may use it in their apps.

**Pull request closed without merging**